apps/camera/test/unit/controllers/confirm_test.js

```diff
@@ -1,8 +1,6 @@
 /*jshint maxlen:false*/
 'use strict';
 
-require('/shared/js/l10n.js');
```

Comment:
I'm just removing three requires in camera because they are not needed, I forgot to remove them in my camera patch, and I don't want to file a separate bug for that so I use the excuse that camera is part of media apps :)